### PR TITLE
Make addon usable as a library

### PR DESCRIPTION
**Pull request by dantman**
This change removes any usage of context.scene.psk_import settings from within ps[ka]import, instead passing them as args.

This allows ps[ka]import to be used from python scripts without actually installing the addon.

### Why?

I'm bulk converting a few hundred models from .psk(x) to .fbx (and also fixing the scale and rotation to import them into Unity).

Because it seems impossible to do otherwise, I've written a python script responsible for doing the batch import, modify, export. However because this is impossible to do with blender's ops functionality ().

Notably one reason for not just having the user install the addon (besides having to tell others in the same community to manually install an extra library in addition to my bulk import/export) is so that I can switch between the normal befzz and no_anim versions for different types of models.

### CODE REVIEW

**Comment by Befzz** [comment]
added to Collaborators, so you can fix what you need.  
pull request looks good.  
but I don't have time to check it on my own ;)

SO. **If it works for you -> feel free to make changes.**

**Comment by ArchUser87** [comment]
Hi dantman, this sounds like something I've wanted to do also, have hundreds of psk(x, a) files I want to convert to FBX, since I'm a noob how do I actually do this with the changes you made? Thx.

**Comment by dantman** [comment]
@ArchUser87 You'll need to write a script to loop over your files, call pskimport and call a similar export method in the fbx addon. Python import doesn't work well for cross-addon import so you'll need to include your __init__.py and a local copy of both addons in your addon and put them all into a .zip file to install and manually use the import utils to import them.

File: addons/io_import_scene_unreal_psa_psk_befzz_no_anim.py [hunk 1]
```diff
@@ -212,7 +212,7 @@ def util_gen_name_part(filepath):
     return re.match(r'.*[/\\]([^/\\]+?)(\..{2,5})?$', filepath).group(1)
 
 
-def pskimport(filepath, bImportmesh, bImportbone, bDebugLogPSK, bImportsingleuv):
+def pskimport(filepath, bImportmesh, bImportbone, bDebugLogPSK, bImportsingleuv, bonesize, bonesize_auto):
```
Comment (Befzz) [inline addons/io_import_scene_unreal_psa_psk_befzz_no_anim.py]: argument **filepath**  
but executing with **filename**

**Comment by ArchUser87** [comment]
Yeah that just went through my head lol, no nothing on python scripting, and for all coding for that matter, barely scratched the surface....

**Comment by Befzz** [comment]
fixed time ago.

**Pull request closed without merging**